ResearchKit/Charts/ORKGraphChartView.h

```diff
@@ -106,20 +109,6 @@ ORK_AVAILABLE_DECL
 - (NSInteger)graphChartView:(ORKGraphChartView *)graphChartView numberOfPointsForPlotIndex:(NSInteger)plotIndex;
```

Comment:
Do we want to take this opportunity to rename `numberOfPointsForPlotIndex` to `numberOfAxesForPlotIndex`? Because it makes more sense to three types of graph.